src/ServiceControl/HeartbeatMonitoring/LegacyHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace ServiceControl.HeartbeatMonitoring
```

Comment:
Yeah. Otherwise they'll go through FLR/SLR. Every event in SC is handled anyway by the global handlers, it's only for commands. These commands don't really make a lot of sense after a restart and can cause problems when trying to ingest them in parallel anyway leading to more congestion.